cmd/version.go

```diff
@@ -51,19 +51,16 @@ func newVersionCmd(homedir, getIstioVersion string) *cobra.Command {
 
 			logger.Infof("getistio version: %s\nactive istioctl: %s\n", getIstioVersion, cur.ToString())
 			k8sCLient, err := util.GetK8sClient()
-			hasK8sErr := false
 			if err != nil {
 				logger.Infof("no active Kubernetes clusters found\n")
-				hasK8sErr = true
 			} else {
 				_, err := k8sCLient.ServerVersion()
```

Comment:
no, you must user `_, err = ` instead of `:=`. This is something you did in the earlier commit.